sdk/storage/Azure.Storage.Queues/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
check which of these exclusions are still needed given the new analyzer package.